### DISCUSSION TITLE
Internal Bluscsi Click Issue Akai Sampler

### DISCUSSION THREAD

**Discussion by diggerjam** [erichelgeson/BlueSCSI-v1#220] [General]
Hi Everyone, 
Just reaching out for some help with an issue with using an internal bluescsi with an Akai 3200xl sampler. I have created hardrive and CD images, and the sampler has no problem loading and saving. However on retrieval of the samples there are clicks in the audio of the sample, sounds like pops on a lp record. It is not the sampler at faults as I can load and save without issue with floppy all zip drive. I had a look at all solder points, and they look fine. Has anyone got any suggestions? It seems like in the communication process these strange  clicks are been added to the sample.
Thanks

**Comment by erichelgeson** [comment]
BlueSCSI writes the bits it gets - can you compare 2 samples, one good one with "clicks" and see if there is a pattern? maybe one of the data lines is making a bad connection.

**Reply by diggerjam** [reply]
Yes, will do and report back, I do know that the clicks/spikes appear as square like waves.

**Reply by diggerjam** [reply]
So on loading the same sample multiple times (the sample is 5 sec in length) the clicks are quite random, sometimes only 3 or 4, other times greater than 10.
